src/my_app/test/counter/cubit/counter_cubit_test.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:bloc_test/bloc_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:very_good_wearos_app/counter/counter.dart';
+
+void main() {
+  group('CounterCubit', () {
```

Comment:
Should we encourage this?

```suggestion
  group('$CounterCubit', () {
```

Reply:
why?